torch/onnx/utils.py

```diff
@@ -495,6 +495,19 @@ def unpack_quantized_tensor(value):
         return (value,)
 
 
+def _pre_trace_quant_model(model, args):
+    r"""Returns `torch.jit.trace(model, args)` if model is quantized. Otherwise do nothing and return
+    original model.
+
+    This is because the quantized weights are stored in special `PackedParams` structure. They need to
+    be handled by `torch.jit.trace` to be exportable to ONNX.
```

Comment:
Please add a link to an issue that we're using to track getting rid of the need for this.

I guess https://github.com/pytorch/pytorch/issues/47887 ?

Reply:
#47887 is a different issue. The closest tracking one is #75761. I will link it here and leave it open then.